development/port-forwards

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+###
+# Use for local development to utilize in cluster service addresses rather then Route objects
+###
+
+set -e
+
+for ARG in $@;
+do
+    if [[ "$ARG" == "-h"* ]] || [[ "$ARG" == "--h"* ]]
+    then
+        echo "Setup port forwarding for local development using internal svc"
+		echo "usage ${0##} context (to execute oc commands as). This is useful if you are running the actual operator the MUO service account [OPTIONAL]"
+        echo "example: ${0##} default/dofinn-20210802/dofinn.openshift" 
+		echo
+        exit 1
+    fi 
+done
+
+OC=$(which oc)
+
+if [[ ! -z ${1+x} ]]
+then
+	OC="$(which oc) --context="$1""
+fi
+
+# Setup for local resolution
+grep 'prometheus' /etc/hosts > /dev/null
+if [[ $? == 1 ]]
+then
+	sudo -- sh -c "echo 127.0.0.1 prometheus-k8s.openshift-monitoring.svc.cluster.local alertmanager-main.openshift-monitoring.svc.cluster.local >> /etc/hosts"
+fi
+
+# Setup prometheus and alertmanager port-forwards
+while true; do $OC port-forward -n openshift-monitoring svc/prometheus-k8s 9091:9091;done &
```

Comment:
why need to run as a loop here?

Reply:
The connections are not stable. If they are not ran like this the upgrade will be disrupted.